public/js/post/post.js

```diff
@@ -109,11 +141,15 @@ var MultipleCourseCode = React.createClass({
 
         return (
             <div id={this.props.courseID} className={classes}>
-                <p className="code" onClick={this.toggleFullInfo}> {this.props.data.categoryName} </p>
+                <p className="code" onClick={this.toggleFullInfo}> {this.props.categoryName} </p>
                 <div id = {'spec' + this.props.courseID.substring(5, this.props.courseID.length)} className={infoClasses}>
                     <p className="full_name"> 
-                        {Array.apply(0, Array(this.props.data.textBoxNumber)).map(function (x, i) {
-                            return <input type='text' defaultValue={me.props.data.courses[i]} onKeyDown={me.handleKeyDown} />;
+                        {Array.apply(0, Array(this.props.textBoxNumber)).map(function (x, i) {
+                            if (me.props.textboxesDisabled) {
+                                return <input type='text' defaultValue={me.props.courses[i]} onKeyDown={me.handleKeyDown} disabled />;
```

Comment:
Cool! I was only skeptical since the link I quoted above actually is a React Issue where they discuss making people use the bare disable attributes instead of the "disabled=true, disabled=false" functionality. I do agree using it for now is cleaner.